src/components/Settings/Settings.tsx

```diff
@@ -44,32 +45,12 @@ export class Settings extends Component<Props, State> {
     cameraDevicesInfo: MediaDeviceInfo[];
     audioDevicesInfo: MediaDeviceInfo[];
     menuBoxRef: Element | undefined;
-    mainMenuRef: HTMLElement | undefined;
 
     constructor(props: Props) {
         super(props);
 
-        let selectedCamera: any = false;
-        if (props.selectedCameraDevice) {
-            selectedCamera = {
-                kind: "videoinput",
-                label: props.selectedCameraDevice.label
-            };
-        }
-        let selectedAudio: any = false;
-        if (props.selectedAudioDevice) {
-            selectedAudio = {
-                kind: "audioinput",
-                label: props.selectedAudioDevice.label
-            };
-        }
         this.state = {
-            isOpen: false,
-            selectedCamera: selectedCamera,
-            selectedAudio: selectedAudio,
-            showAudioSettings: false,
-            showCameraSettings: false,
-            showScreenShareSettings: false,
+            showSettingsOf: false,
```

Comment:
```suggestion
```
if no settings should be visible on the first render, then remove it from the `constructor`.

Reply:
done